lib/interactive.py

```diff
@@ -250,25 +243,16 @@ def plot_route(self, messageId):
                             else:
                                 msgType = "Forwarding message"
```

Comment:
I noticed that actually this happened in the previous PR, but for me there's no space between those words anymore. I used to write `msgType = "Forwarding\/message"`, for which it does work.